src/enums/endpoints.ts

```diff
@@ -8,7 +8,8 @@ enum endpoints { // TODO: rename as endpointPath
     ROUTEPATH = 'routePath',
     ROUTELINK = 'routeLink',
     NODE = 'node',
-    LINK = 'link'
+    LINK = 'link',
+    KILPI_VIA = 'kilpivia'
```

Comment:
If having KILPI_VIA on its own endpoint, there will be a problem: if routePath saving fails, also kilpiVia save should be rolled back and vise versa. That one will be hard / unpractical to implement with separate API calls

I think you should save kilpiVia's with same API call routePath so that you can make those savings depend on each other

(also kilpi_via saving is strongly related to saving routePath so thats why it also belongs to the same API call)